packages/react-bindings/src/styles/calculateAnimationTimeout.ts

```diff
@@ -0,0 +1,10 @@
+import convertCssTimeToNumber from './convertCssTimeToNumber'
+import * as _ from 'lodash'
+
+const calculateAnimationTimeout = (duration: string, delay?: string): number => {
+  if (_.isNil(duration) || duration === 'inherit' || duration === 'initial') return 0
```

Comment:
According to types `duration` can be only string

A.  Update typings

```tsx
const calculateAnimationTimeout = (duration: string | undefined, delay?: string): number => {
```

B. Remove `_.isNil()`

```suggestion
  if (duration === 'inherit' || duration === 'initial') return 0
```